app/timetables/admin.py

```diff
@@ -97,7 +97,7 @@ class EventAdmin(TimeStampAdmin):
 class VendorAdmin(DefaultAdmin):
     """Admin customisation for Vendor model."""
 
-    fields = ('name', 'slug', 'info', 'start_date', 'end_date')
+    fields = ('name', 'slug', 'info')
```

Comment:
It's because VendorService model models the tenure of a vendor for a timetable; so VendorService is more appropriate to track the start and end dates.